include/circt/Dialect/Ibis/IbisOps.td

```diff
@@ -160,13 +167,48 @@ def CallOp : IbisOp<"call", [CallOpInterface]> {
   }];
 }
 
+def GetParentOp : IbisOp<"get_parent"> {
```

Comment:
"Parent" is ambiguous. Does it mean in the static class hierarchy (which would only make sense in a container) or in the instance hierarchy.

Reply:
Attempted to disambiguate.